imagetest/test_suites/image_boot/shutdown_test.go

```diff
@@ -0,0 +1,31 @@
+package imageboot
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+const shutdownTime = 110 // about 2 minutes
+
+// TestGuestShutdownScript test that shutdown scripts can run for around two minutes
+func TestGuestShutdownScript(t *testing.T) {
+	_, err := os.Stat(markerFile)
+	if os.IsNotExist(err) {
+		// first boot
+		if _, err := os.Create(markerFile); err != nil {
+			t.Fatalf("failed creating marker file: %v", err)
+		}
+		t.Fatal("marker file does not exist")
+	}
+	// second boot
+	bytes, err := ioutil.ReadFile("shutdown.txt")
```

Comment:
no path means in current directory. every run is in its own directory. as with our other reboot tests, extract the file path to a constant and put it in / or /root/